packages/react/src/hooks/useManageSubscription.ts

```diff
@@ -39,6 +43,17 @@ export const useManageSubscription = (
   const [isUnsubscribing, setIsUnsubscribing] = useState(false);
 
   useEffect(() => {
+    if (w3iClient) {
+      console.log(
+        ">>>>1 stting sub",
+        w3iClient.getSubscription(account, domain)
+      );
+      setSubscription(w3iClient.getSubscription(account, domain));
+    }
+  }, [clientLoading, w3iClient]);
```

Comment:
Does the `clientLoading` playing a role here? Should we make a conditional check like "if clientLoading is true" etc. Is it just retrigger the function when `clientLoading` value changed?

Reply:
I thought that initially but we'd still need to depend on w3iClient. That was the main bug

Reply:
> That was the main bug

Oh, got it. So how would were you able to test it (since we're still have a bug on subscriptions)? :D